### ISSUE TITLE
adding altIdentifier

### ISSUE THREAD

**Issue by emylonas** [Brown-University-Library/usep-data#165]
I have a note about adding the option of inserting an altIdentifier in the msIdentifier. Perhaps for inscriptions that change ID or location. If this is still something we want, we should describe it in more detail. Also this is something that will require Author framework changes - to add and display the field.

**Comment by jbodel** [comment]
Yes, this will be useful to have. How much detail do you think we need? There are so many reasons (some internal to USEP, some not) why an inscriptions might be known by two different IDs that I'm not sure we could list them all. Would it be enough simply to say something like "Alt ID is used in cases in which an inscription already assigned a USEP number subsequently becomes identified by a different designation"? In cases where an item has already had several different inv. or acc. nos internally (e.g. the JHU collection), we have used the most up-to-date one at the time of registration. Sometimes an item IDed by us with a "Tmp" designation is subsequently assigned its own local accession no. In these cases it is (obviously) useful to record both our old and the new nos.

**Comment by sdigiulio** [comment]
Following up on this after a long delay: yes, I think what you've suggested here sounds fine.
